src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -97,4 +94,57 @@ private void setMetadata(Feature[] features, String jsonFile, int jsonFileNo) {
             feature.setMetaData(jsonFile, jsonFileNo, configuration);
         }
     }
+
+    /**
+     * Parses passed properties files and adds metadata to overview-features page
```

Comment:
nice, remember about dot at the end of the first sentence.

Reply:
Cool thanks I'll fix this too